packages/calcite-components/src/components/block-section/block-section.tsx

```diff
@@ -259,13 +259,7 @@ export class BlockSection implements LocalizedComponent, T9nComponent, LoadableC
     return (
       <Host>
         {headerNode}
-        <section
-          aria-expanded={toAriaBoolean(open)}
-          aria-labelledby={buttonId}
-          class={CSS.content}
-          hidden={!open}
-          id={regionId}
-        >
+        <section aria-labelledby={buttonId} class={CSS.content} hidden={!open} id={regionId}>
```

Comment:
Not to the full context of the PR, but adding the following to the `section` provide additional context with JAWS when the section's content contains text and is expanded:
```ts
aria-live="polite"
role="region"
```

Reply:
Per our Teams convo, and Franco's sleuthing, [`section` inherits a region `role`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#technical_summary), so the `role` is not needed in addition to `aria-live`.